instructure/canvas-ios
# improve horizontal menu

refs: MBL-13196
affects: Parent
release note: none

test plan:

student:
1. navigate to course syllabus that has
  a. no syllabus (should only display assignments, no tabs )
  b. with syllabus (there should be two tabs)

Teacher
1. navigate to speedgrader list of students, click eye
2. make sure there are 2 tabs and you can navigate between tabs w/ iphone / ipad
